akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -523,7 +526,8 @@ private[akka] class ShardRegion(
     if (before != after) {
       if (log.isDebugEnabled)
         log.debug(
-          "Coordinator moved from [{}] to [{}]",
+          "{}: Coordinator moved from [{}] to [{}]",
+          typeName,
           before.map(_.address).getOrElse(""),
```

Comment:
Might this we better placement? I'm guessing for humans vs machines reading logs, it helps for the shard typeName to lead the line, but it offers less context at the start.
```scala
log.debug("Coordinator for Shard [{}] moved from [{}] to [{}]",  typeName, from, to)
```

Reply:
I tried to make all logs to look similar - with `${typeName}: ` as prefix

Reply:
@helena but it's not a `Shard`, `shard` and `typename` are different things

Reply:
Correct, I suggested based on the existing
```scala
Message does not have an extractor defined in shard [{}]
```

Reply:
@helena I'm fine with suggestion, however I'd rather change 
```scala
log.warning("Message does not have an extractor defined in shard [{}] so it was ignored: {}", typeName, unknownMsg)
```

to 

```scala
log.warning("{}: Message does not have an extractor defined in shard, so it was ignored: {}", typeName, unknownMsg)
```
So not it is consistent. @helena what do you think ?

Reply:
@helena actually I'd like to have asap, the actual order of words does not really matter much :)

So please propose the format I could re-use everywhere and I could update the pr there after.